src/open_inwoner/haalcentraal/tests/test_logging.py

```diff
@@ -13,16 +13,16 @@
 
 @freeze_time("2021-10-18 13:00:00")
 @requests_mock.Mocker()
+@override_settings(
+    LOG_OUTGOING_REQUESTS_ENABLED=True,
+    LOG_OUTGOING_REQUESTS_DB_SAVE=True,
+)
 class TestPreSaveSignal(ClearCachesMixin, HaalCentraalMixin, TestCase):
```

Comment:
- I don't think there is a setting `LOG_OUTGOING_REQUESTS_ENABLED`
- `LOG_OUTGOING_REQUESTS_DB_SAVE=True` is the defaul, so override shouldn't be necessary

Reply:
Good catch, I see this got updated since I last had overrides in my local.py

I'm keeping the `@override_settings(LOG_OUTGOING_REQUESTS_DB_SAVE=True)` here because settings like these are often modified in local.py (as I do).